### DISCUSSION TITLE
How does one build WiX 3.11.2?

### DISCUSSION THREAD

**Discussion by sszeller** [wixtoolset/issues#6577] [Questions]
I downloaded the source code, loaded up the solutions, installed the C++ necessaries for Visual Studio 2019, and tried to build. I received this error:

Strong name validation for the WiX toolset must be disabled on a developer machine to build. Execute 'msbuild C:\Users\scott.zeller\Downloads\wix3-wix3112rtm\tools\OneTimeWixBuildInitialization.proj' from an elevated command prompt.	Resources			

So, I hunted that file down, tried executing, and it couldn't find sn.exe due to a bad registry path. Installed 4.7.2 Framework SDK (again... somehow the registry got messed with), edited the .proj file, and ran it. This time it ran successfully. However, I used the same sn.exe (32-bit) for both the x64 and x86 lines in the .proj file (couldn't find another sn.exe). Is that a problem?

Restarted Visual Studio and am still getting the same message as before.

What am I missing?

**Comment by BMurri** [comment]
Does anything on [this page](https://wixtoolset.org/development/building-wix/) help?

**Reply by sszeller** [reply]
I'm pretty sure I worked through that. I've moved on to a different project, so I won't be following up on this.
